boolean_algebra/nandgate.py

```diff
@@ -0,0 +1,57 @@
+"""
+The NAND gate (negated AND) gives an output of 0 if both inputs
+are 1, it gives 1 otherwise.
+
+Below is the truth table for NAND gate.
+
+| Input 1 | Input 2 | Output |
+| ------- | ------- | ------ |
+|    0    |    0    |   1    |
+|    0    |    1    |   1    |
+|    1    |    0    |   1    |
+|    1    |    1    |   0    |
+
+Code By: @rajatshenoi
+"""
+
+
+def nand_gate(input1: int, input2: int) -> int:
+    """
+    >>> nand_gate(0, 0)
+    1
+    >>> nand_gate(0, 1)
+    1
+    >>> nand_gate(1, 0)
+    1
+    >>> nand_gate(1, 1)
+    0
+    >>> nand_gate(0.0, 0.0)
+    1
+    >>> nand_gate(0, -7)
+    1
+    """
+
+    if input1 == input2 == 1:
+        return 0
+    else:
+        return 1
```

Comment:
```suggestion
    return 1
```
using an else here is unnecessary